index.bs

```diff
@@ -1106,9 +1126,11 @@ input parameters:
 - The <a>clientDataHash</a>, which is the hash of the serialized {{ClientData}} and is provided by the client.
 - The {{Account}} information provided by the [RP].
 - The {{ScopedCredentialType}} and cryptographic parameters requested by the [RP], with the cryptographic algorithms normalized
-    as per the procedure in [[WebCryptoAPI#algorithm-normalization-normalize-an-algorithm]].
+    as per the procedure in [[WebCryptoAPI#algorithm-normalization-normalize-an-algorithm]]
 - A list of {{ScopedCredential}} objects provided by the [RP] with the intention that, if any of these are known to the
-    authenticator, it should not create a new credential.
+    authenticator, it should not create a new credential. 
+- A boolean called {{ScopedCredentialOptions/verificationGesture}} provided by the [RP] with the intention that, if the parameter 
+    is set to true, the authenticator SHOULD perform a user verification process during the operation.
```

Comment:
s/user verification/[=user verification=]/

i wonder whether this would be a SHOULD or a MUST